ppafm/ocl/AFMulator.py

```diff
@@ -457,7 +457,6 @@ def quick_afm(file_path, scan_size=(16, 16), offset=(0, 0), distance=8.0, scan_s
             QZs = [0, 0, 0, 0]
         elif tip == 'pz':
             Qs = [10*charge, -10*charge, 0, 0]
-            Qzs = [0.1, -0.1, 0, 0]
```

Comment:
I am not quite sure, if you wast to remove this line @NikoOinonen - could you look at it?

Reply:
I missed this before. Yeah, this does not look right. Why would it remove this line?

Reply:
yes, I also think this is wrong (should not be removed)